spec/controllers/Api/v1/users_controller_spec.rb

```diff
@@ -0,0 +1,57 @@
+module Api
+  module V1
+    describe UsersController, type: :controller do
+      context "#show" do
+        let!(:first_user) {User.create(
+          email: "menternshift@gmail.com",
```

Comment:
Not an issue with this specific PR, but we should probably look into using a gem like `factory_bot` https://github.com/thoughtbot/factory_bot, it lets us generate object instances with default values very easily, eliminates lots of boilerplate.